packages/datadog-plugin-pg/src/index.js

```diff
@@ -36,9 +38,14 @@ function createWrapQuery (tracer, config) {
 
       const originalCallback = pgQuery.callback
       const statement = pgQuery.text
+      const values = pgQuery.values
 
       span.setTag('resource.name', statement)
 
+      if (config.includeQueryParams !== undefined) {
+        span.setTag('sql.params', config.includeQueryParams(values))
```

Comment:
I didn't realize this when proposing to add a filter function, but what if params are filtered? Since it's just an ordered array, wouldn't that mean that you would end up with a different number of placeholders and values? How would you make sense of the result then? Maybe it would be simpler to just have a boolean as you originally proposed. Sorry for the back and forth on this, I didn't realize that the params were just an array of values at first, I assumed it was key/value pairs.

Reply:
Or maybe could it actually be a key/value pair? It looks like the params are numerically identified.

Reply:
It's just an array of values. Number id is actually an index in the array.